### DISCUSSION TITLE
Date format?

### DISCUSSION THREAD

**Discussion by Petruchio96** [FuLoRi/Analyze-LastPassVaultGUI#11] [Q&A] [answered]
Just tried your script after listening to Steve on Security Now, thanks for providing this for us! One question, what is the date format for the "Last Touched" field? I have dates listed like '1562184788' in my csv file and I don't know what format that is in.

**Comment by FuLoRi** [comment, marked as answer]
You're welcome! :) I suspect it's a UNIX epoch timestamp. There's a web-based converter here: https://www.epochconverter.com/.

**Comment by Petruchio96** [comment]
That's exactly what it is, thanks! I completely forgot about Unix Epoch time, it's been a few years (decades?) that I've messed with that.

**Reply by FuLoRi** [reply]
FWIW, the latest version of the script now converts the Unix Epoch into a human-readable date/time string, thanks to @Iczer1.
